spec/spec_helper_acceptance_local.rb

```diff
@@ -10,5 +10,9 @@
   c.before :suite do
     # Download the plugins to ensure up-to-date facts
     PuppetLitmus::PuppetHelpers.run_shell('/opt/puppetlabs/bin/puppet plugin download')
+    # Put a test PE license in place
+    PuppetLitmus::PuppetHelpers.run_shell('echo -e "#######################\n#  Begin License File #\n#######################\n \n# PUPPET ENTERPRISE LICENSE - test
```

Comment:
Test is failing due to puppet run having unexpected changes, manually when I put a license file in place and run Puppet the change is `Notice: /Stage[main]/Puppet_enterprise::License/File[/etc/puppetlabs/license.key]/seluser: seluser changed 'unconfined_u' to 'system_u'` so this will need to be accounted for.

Reply:
The issue with my manual testing is the start date in the fake license file is after today's date so that will need to be changed for the tests to pass. After editing the dates my manual testing all worked.

Reply:
Thank you Elaine, missed this in haste, will update with a fresh commit